mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -272,6 +273,33 @@ def test_ascii(tmpdir):
     assert_allclose(times_new, times)
 
 
+def test_ch_names_comma():
+    """Test that channel names containing commas are properly read."""
+    find_line = "Ch4=F4,,0.5,µV"
+    change_to = r"Ch4=F4\1foo,,0.5,µV"  # commas in BV are encoded as \1
+
+    # Manipulate existing vhdr file to contain a channel with comma
+    with open(vhdr_path, 'r') as fin:
+        lines = fin.readlines()
+    new_lines = []
+    for line in lines:
+        if find_line in lines:
+            new_lines.append(change_to)
+            continue
+        new_lines.append(line)
+
+    # Write temporary vhdr file for the test
+    tmpvhdr = vhdr_path.replace('.vhdr', '_comma-test.vhdr')
+    with open(tmpvhdr, 'w') as fout:
+        fout.writelines(lines)
+
+    raw = read_raw_brainvision(tmpvhdr)
+    os.remove(tmpvhdr)
```

Comment:
no need for this, `pytest` should clean up on its own

Reply:
even without a fixture or anything?

Reply:
Oh sorry I assumed you were using a `tmpdir` fixture. You definitely should use `tmpdir` and `shutil.copyfile` instead of writing anything to the `vhdr_path` directory